Kinvey.Core/Core/KinveyClientRequestInitializer.cs

```diff
@@ -122,10 +129,12 @@ public void Initialize<T>(AbstractKinveyClientRequest<T> request)
             {
                 credential.Initialize(request);
 			}
-			if (request.RequireAppCredentials)
+
+            if (request.RequireAppCredentials)
             {
-                request.RequestAuth = new HttpBasicAuthenticator(AppKey, AppSecret);
+                request.RequestAuth = new HttpBasicAuthenticator(AuthServiceID, AppSecret);
```

Comment:
shouldn't this be "`AppKey`.`AuthServiceID `"?

Reply:
If you see in line 116, the `clientId` actually has this form of `AppKey.MicID`, so it is already incorporated.